framework/tools/generators.py

```diff
@@ -27,23 +27,56 @@ def generate_string(length: int, additional_characters: list = None) -> str:
     return "".join(result)
 
 
-def generate_user(password: str = DEFAULT_PASSWORD) -> dict:
-    """Generating a user with the specified password
+def generate_user(
+    first_name_length: int = None,
+    last_name_length: int = None,
```

Comment:
I suggest not changing the data types. What do you think?

Reply:
Fixed.